httpReq.test.js

```diff
@@ -0,0 +1,60 @@
+import test from 'ava';
+import fetch from 'node-fetch';
+import httpReq from './httpReq';
+
+const baseUrl = 'https://httpbin.org';
+
+test.before(t => {
+    window.fetch = fetch;
+    global.dw = { backend: { __api_domain: 'api.datawrapper.local' } };
+});
```

Comment:
You could consider stubbing fetch in order to speed up and further isolate the test (see [fetch.test.js](https://github.com/datawrapper/shared/blob/master/fetch.test.js) for an example of how to do that.

Reply:
I did consider but I didn't like the fact that I have to mock the responses. If so I would want a real "echo" server, ideally one that behaves like httpbin. In theory we could fire up [httpbin](https://github.com/postmanlabs/httpbin) itself but it's written in Python and I didn't want to throw that into our local test setup. There are some similar echo-servers in node but I wasn't really impressed by any of them so far. Also considered writing one myself but decided it would be out of scope for this PR.

Reply:
That makes perfect sense.